package.json

```diff
@@ -32,8 +32,8 @@
   "dependencies": {
     "core-js": "^3.16.4",
     "grimoire-kolmafia": "^0.3.18",
-    "kolmafia": "^5.27578.0",
-    "libram": "^0.8.9"
+    "kolmafia": "^5.27609.0",
```

Comment:
If we are using an updated version of KoLMafia, then we should also require that users have at least that version (where we enforce this with `sinceKolmafiaRevision()` in `main.ts`)